### ISSUE TITLE
NTR, pulmo: pulmonary Amyloidosis

### ISSUE THREAD

**Issue by GrieseAG** [obophenotype/human-phenotype-ontology#6452]
**Preferred term label:**
pulmonary Amyloidosis

**Synonyms**
extracellular deposit of amyloid in the lung

**Definition (free text, please give PubMed ID)**
There are three main patterns of pulmonary involvement:
tracheobronchial, nodular parenchymal and diffuse alveolar-septal amyloidosis. Amyloidosis may be divided into primary and secondary (associated with chronic inflammatory diseases). Amyloidosis may also be divided into localized or generalized (in addition to the parenchyma, amyloidosis may also affect the pleura, the pulmonary arteries, the hilar and mediastinal lymph nodes, and the diaphragm). 

Comment: The basic alteration is the extracellular deposit of amyloid L (AL) (primary form) or amyloid A (AA) (secondary form). Amyloid is a protein-derived substance which deposits in the extracellular spaces. It appears homogeneous and slightly eosinophilic in H&E staining, positive to Congo red staining, exhibiting a green birefringence under polarized light and fluorescent after thioflavin staining

Book: Diffuse lung diseases, Clinical Features, Pathology, HRCT. M. Maffessanti & G. Dalpiaz. ISBN-13 0978-88-470-0429-0

**Parent term (use hpo.jax.org/app)**
Amyloidosis HP:0011034

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
The deposit of amyloid material in the lungs may be associated with several diseases affecting the lungs (chronic tuberculosis, bronchiectasis, pulmonary abscess, chronic aspergillosis, rheumatoid pleuritis, extrinsic allergic alveolitis, fibrosis) or other organs (Crohn’s disease, Hodgkin’s disease, renal carcinoma)

**Your nano-attribution (ORCID)**
lmu: chrapp, @mgriese, @kknoflach

**Comment by pnrobinson** [comment]
This is the definition for cardiac amyloidosis.
Would it be good to adapt this for the lung?

Extracellular deposition in cardiac tissue of a proteinaceous material that, when stained with Congo red, demonstrates apple-green birefringence under polarized light and that has a distinct color when stained with sulfated Alcian blue. Viewed with electron microscopy, the amyloid deposits are seen to be composed of a beta-sheet fibrillar material. These nonbranching fibrils have a diameter of 7.5 to 10 nm and are the result of protein misfolding.

Also, the term should also have a parent term in the lung subhierarchy? Would this be interstitial?

**Comment by pnrobinson** [comment]
-- interstitial +1
-- definition +1

**Comment by pnrobinson** [comment]
added
